x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/components/agent_policy_form.tsx

```diff
@@ -284,7 +284,7 @@ export const AgentPolicyForm: React.FunctionComponent<Props> = ({
           }}
         />
       </EuiDescribedFormGroup>
-      {isEditing && 'id' in agentPolicy && agentPolicy.is_managed !== true ? (
+      {isEditing && 'id' in agentPolicy && !agentPolicy.is_managed && !agentPolicy.is_default ? (
```

Comment:
I think we should hide this for the default fleet server too no? 
```suggestion
      {isEditing && 'id' in agentPolicy && !agentPolicy.is_managed && !agentPolicy.is_default && !agentPolicy.is_default_fleet_server ? (
````

Reply:
++ looking at the server code we don't allow `is_default_fleet_server` to be deleted either, so we should do same on UI